### ISSUE TITLE
RangeError for long URLS

### ISSUE THREAD

**Issue by wtn** [peterc/webloc#3]
When trying to write a webloc with a long URI (247 characters) I got the following error:
```
webloc-0.2.0/lib/webloc.rb:55:in `chr': 265 out of char range (RangeError)
	from webloc-0.2.0/lib/webloc.rb:55:in `data'
	from webloc-0.2.0/lib/webloc.rb:61:in `block in save'
	from webloc-0.2.0/lib/webloc.rb:61:in `open'
	from webloc-0.2.0/lib/webloc.rb:61:in `save'
```

This issue appears to be distinct from #1.

**Comment by peterc** [comment]
Hmm, curious. The README says URLs up to 255 characters (and there's a technical reason for this - though one that could be overcome) but 247 should be fine. I'll take a look!

**Comment by peterc** [comment]
Okay, 237 seems to be the actual limit. I'll fix the README first before looking into exactly why this is.

**Comment by peterc** [comment]
I got more intimate with the binary PLIST format than I wanted to there, but I have extended webloc to support up to two bytes for URL length which safely gets us up to 2048 characters. I've added a test to read and write a 2048 character URL and all tests are now passing once again. Thanks for the report! Consider this fixed in the 0.3.0 release just pushed! :)
